index.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <<meta name="viewport" content="width-device-width, initial-scale=1.0">
+    <meta name="viewport" content="width-device-width, initial-scale=1.0">
     <title>Welcome to Master/title>
   </head>
 <body>
+	<h1>Welcome to Master! Enjoy your stay</h1>
```

Comment:
Could you remove the 'Enjoy your stay. string and make the 'Welcome to Master!" string red?